cfglib.py

```diff
@@ -211,7 +211,17 @@
     cfg.BoolOpt('block_migration', default=False,
                 help='live-migration without shared_storage'),
     cfg.StrOpt('host_eph_drv', default='-',
-               help='host ephemeral drive')
+               help='host ephemeral drive'),
+    cfg.StrOpt('connection', default='mysql+mysqlconnector',
+               help='driver for db connection'),
+    cfg.StrOpt('host', default='',
```

Comment:
I would recommend renaming option names to have `db_` prefix, for example, `db_host` instead of `host`, `db_user` instead of `user` and so on. I would also recommend renaming the rest of DB options in other config groups according to that pattern, it is much more readable, IMHO.